Tests/Fixtures/Entity/ContainerEntity.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Fixtures\Entity;
+
+class ContainerEntity
```

Comment:
@Soullivaneuh can you please answer the question?

After that i would like to merge this PR.
Thank you